bandit/formatters/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Bandit is a tool designed to find common security issues in Python code.
+
+Bandit is a tool designed to find common security issues in Python code.
+To do this Bandit processes each file, builds an AST from it, and runs appropriate
+plugins against the AST nodes. Once Bandit has finished scanning all the files
+it generates a report.
+
+https://bandit.readthedocs.io/
+"""
```

Comment:
Even though not strictly documented in a coding conventions doc, we don't normally put docstrings in __init__.py files... unless there is code within it.

Reply:
I haven't seen this convention much either.  The linter warned on this.  Should the warning be disabled?  I don't think it hurts to add it, and could add a brief note to see the documentation in the main class.

https://stackoverflow.com/questions/37019744/is-there-a-consensus-what-should-be-documented-in-the-classes-and-init-docst

Reply:
Reverted init.py docstrings